R/geographicmap.R

```diff
@@ -388,6 +388,20 @@ leafletMap <- function(coords, colors, opacity, min.value, max.range, color.NA,
     max.values <- unique(coords$table.max[!is.na(coords$table.max)])
     if (length(max.values) == 1)
         max.values <- c(max.values, max.values * 1.1)
+    
+    # If we are close to the antimeridian, wrap coords and polygons
+    if ("longitude" %in% colnames(coords@data)) {
+        lng <- coords@data$longitude
+        if (any(lng > 170)) {
```

Comment:
Is this `if` necessary? Could we just run the code below all the time?

Reply:
I think you do need it. Most of the time we want the longitude values to stay in [-180, 180]. We don't want to introduce discontinuities around longitude = 0 instead (e.g. Algeria)

Reply:
OK. Would a map of the US be affected by this since there are islands in Alaska where `lng > 170`?

Reply:
No, US longitude ranges from about -166 to -60 (with Alaska in -166 to -145), so it never matches that `lng > 0` condition

Reply:
Ah, I see what you mean about those islands, however, the don't get listed in our map (in R/sysdata.rda), so we can ignore them. I tested it for high resolution maps and its still ok.

Reply:
I've added an additional clause in the if statement so that line 393 matches line 489. That is, it US regions won't get longitudes increased by 360 if setView is fixed to a particular position